src/zoneerder/components/referentielaag-map.scss

```diff
@@ -43,3 +43,9 @@ referentie-laag-map {
     /* adjust the size, position, etc. of the legend for small screens here */
   }
 }
+
+@media all and (display-mode: fullscreen) {
```

Comment:
Is dit nodig? Voor nu zorgt dat nog altijd dat die buttons buiten het scherm vallen 

![image](https://github.com/user-attachments/assets/e930890d-6142-408e-83cb-40619b422cca)

Reply:
dit is enkel van toepassing wanneer fullscreen he. Heb de mapnode ietske minder hoog gemaakt.